components/dashboard/src/Login.tsx

```diff
@@ -67,8 +59,6 @@ export const Login: FC<LoginProps> = ({ onLoggedIn }) => {
         providerFromContext = authProviders.data.find((provider) => provider.host === hostFromContext);
     }
 
-    const showWelcome = !hasLoggedInBefore() && !hasVisitedMarketingWebsiteBefore() && !urlHash.startsWith("https://");
```

Comment:
**thought**: Skipped removing this in https://github.com/gitpod-io/gitpod/pull/17748.